app/presenters/result_set_presenter.rb

```diff
@@ -21,7 +21,11 @@ def initialize(content_item, facets, results, filter_params, sort_presenter, met
   end
 
   def displayed_total
-    "#{number_with_delimiter(total)} #{pluralised_document_noun}"
+    heading_text = "#{number_with_delimiter(total)} #{pluralised_document_noun}"
+    # TODO: to be displayed for all finders pending content + design review. See: https://trello.com/c/bWnyVnhX/1983-update-x-licences-results-text
+    return "results: #{heading_text}" if content_item.document_noun == "licence"
```

Comment:
Let's avoid the bare string for "results" and put that into the translation files for consistency/good practice.

Reply:
There's not too much in the way of translations for finder frontend but i've moved the whole sentence in to the search_result_presenter yaml file (with a few changes required by Rubocop). I've added as a separate commit here but i'll squash in to one commit when approved.